### PR TITLE
RSWEB-7648: add contentOffset to example pages

### PR DESCRIPTION
**Pull request by writes**
Adds ```<div class="contentOffset">``` to example pages on zoolander to account for navigation height at top of page. 

*DOES NOT AFFECT www

ex: 
http://localhost:9000/derek/examples/events-interior
http://localhost:9000/derek/examples/events
http://localhost:9000/derek/examples/homepage
http://localhost:9000/derek/examples/office365
http://localhost:9000/derek/examples/section-overview
http://localhost:9000/derek/examples/swatches

### CODE REVIEW

**Comment by WidgetsBurritos** [comment]
👍

File: styleguide/derek/examples/homepage.ejs [hunk 1]
```diff
@@ -1,257 +1,259 @@
...
+	    <div class="row half-padding-full">
+	      <div class="col-md-11 center-block">
+	      <a href="[rs_link:managed-security]" class="solution solutions-hoverArea">
+	          <div class="col-md-2 col-md-push-1 col-xs-4">
+	              <div class="border-white center">
+	                <i class="rsweb mng-security white rs-4x"></i>
+	              </div>
+	            <h6 class="white center">Managed Security</h6>
+	          </div>
+	        </a>
```
Comment (SyntaxStacks) [inline styleguide/derek/examples/homepage.ejs]: use space in place of tabs

**Comment by SyntaxStacks** [comment]
there's a mix of spaces and tabs in this PR
please convert tabs to spaces

**Comment by writes** [comment]
Hey @SyntaxStacks thanks for the feedback. I have updated the spacing. Let me know if you see anything else.

**Pull request merged**